cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,50 @@
+describe("User Authentication", () => {
+  describe("Login and Profile Access", () => {
+    it("allows the user to log in and access their profile", () => {
+      cy.visit("/");
+
+      // Click login button without changing the default credentials
+      cy.get(".login-btn").click();
+
+      // Wait for the redirection delay
+      cy.wait(3000); // Increased to account for the 2-second delay plus some buffer
```

Comment:
Why wait for this long? You should be intercepting the API request so you know when the async function is complete.
Read the docs!
https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting